docker/go/nethttp/go.mod

```diff
@@ -0,0 +1,17 @@
+module testapp
+
+go 1.11
+
+require (
+	github.com/google/go-cmp v0.3.0 // indirect
+	github.com/kr/pty v1.1.4 // indirect
+	github.com/santhosh-tekuri/jsonschema v1.2.4 // indirect
+	github.com/stretchr/objx v0.2.0 // indirect
+	go.elastic.co/apm v1.3.0
+	go.elastic.co/apm/module/apmhttp v1.3.0
+	golang.org/x/crypto v0.0.0-20190426145343-a29dc8fdc734 // indirect
+	golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c // indirect
+	golang.org/x/text v0.3.2 // indirect
+	golang.org/x/tools v0.0.0-20190503185657-3b6f9c0030f7 // indirect
+	gopkg.in/yaml.v2 v2.2.2 // indirect
+)
```

Comment:
There need to be a couple of "replace" lines like this:

https://github.com/elastic/apm-agent-go/blob/47cc289609d7000794e858f84652c5ddbe17e623/module/apmecho/go.mod#L17-L19

They should point to the location where the apm-agent-go repo is checked out. Without those, `go build` will fetch `go.elastic.co/apm` from GitHub.